src/main/scala/org/broadinstitute/dsde/firecloud/EntityClient.scala

```diff
@@ -368,129 +369,30 @@ class EntityClient (requestContext: RequestContext, modelSchema: ModelSchema, go
 
   def importPFB(workspaceNamespace: String, workspaceName: String, pfbRequest: PfbImportRequest, userInfo: UserInfo): Future[PerRequestMessage] = {
```

Comment:
importPFB is now simplified.  It delegates validation and permission-checking to the import service.